apps/web/i18n/routes.tsx

```diff
@@ -0,0 +1,109 @@
+import { Locale, defaultLanguage } from './I18n'
+
+export interface AnchorAttributes
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  href: string
+  as: string
+}
+
+export type ContentType =
+  | 'article'
+  | 'page'
+  | 'category'
+  | 'contentcategory'
+  | 'articlecategory'
+  | 'news'
+  | 'search'
+  | 'lifeevent'
+  | 'lifeeventpage'
+  | 'adgerdir'
+  | 'vidspyrna-frontpage'
```

Comment:
Looks like some types are missing e.g. vidspyrnapage.
Is the plan to add missing types when we integrate this into the rest of the web?

Reply:
either way works fine to me :) i just moved over the types that were already defined in the project, if you can point out some overview of all types i can add the missing ones in, or add them in one by one